rewriter/initializer.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SORBET_REWRITER_INITIALIZER_H
+#define SORBET_REWRITER_INITIALIZER_H
+#include "ast/ast.h"
+
+namespace sorbet::rewriter {
+
+class Initializer final {
```

Comment:
All other DSL passes have a description of the transformation in the header; can we add one here?